### DISCUSSION TITLE
new placeholder that combines accrued and bonus claim blocks to form total claim blocks.

### DISCUSSION THREAD

**Discussion by NxTPanic** [GriefPrevention/GriefPrevention#1603] [Ideas]
Dear Dev,

Can you  add a new placeholder that combines accrued and bonus claim blocks to form total claim blocks.

Kind regards!

**Comment by Jikoo** [comment]
Again, if you're looking for some change to the GP PAPI integration, you should make requests [where it is maintained](https://github.com/pixar02/Expansions).
